packages/sources/react/rollup.config.js

```diff
@@ -5,24 +5,22 @@ import url from '@rollup/plugin-url';
 import svgr from '@svgr/rollup';
 import external from 'rollup-plugin-peer-deps-external';
 import postcss from 'rollup-plugin-postcss';
-import pkg from './package.json';
 
-const TARGETS = [
-  { format: 'cjs', dir: './dist', tsconfig: './tsconfig.cjs.json' },
-  { format: 'es', file: pkg.module, tsconfig: './tsconfig.es.json' },
-];
```

Comment:
@Shyrro I don't know the previous build very well, but there is no need for that granularity here? i.e. to be able to differentiate a different dist, a different config, etc.

I understand the use of ternary below, but aren't there cases where we would potentially want to export something other than es module or common js?

Reply:
@thibault-mahe Actually, we can do the same and have as much outputs as we wants without this. 

The main difference here is that before, since we used the `TARGETS.map` as export default for our config, this meant that rollup used to run the build command twice with each configuration. Whereas now, we only run the build command once and have the right files exported